### PR TITLE
Chore/misc fixes

### PR DESCRIPTION
**Pull request by iccicci**
# Context

- Our CI sometimes suffers of some flakiness
- CI containers logs are dumped in case of job cancel (i.e. when CI fails due to timeouts)

# Proposed Solution

- Increased txs submit timeout in _local-network_ startup.
- Changed the condition to run job step which dumps the logs

### CODE REVIEW

**Review by mkazlauskas — APPROVED** [review]
Looks good, thanks @iccicci !

**Review by ginnun — APPROVED** [review]
lgtm

File: packages/e2e/local-network/scripts/mnemonic_keys.sh
Comment (mirceahasegan) [inline packages/e2e/local-network/scripts/mnemonic_keys.sh:1]: I don't see any changes to the file

Comment (iccicci) [reply packages/e2e/local-network/scripts/mnemonic_keys.sh:1]: There was no changes in the content of the script, only in its permissions

**Review by mirceahasegan — APPROVED** [review]
Great work, @iccicci 🚀

**Pull request merged**